ALICE3/Core/DelphesO2TrackSmearer.cxx

```diff
@@ -231,6 +231,7 @@ double TrackSmearer::getPtRes(int pdg, float nch, float eta, float pt)
   auto val = sqrt(lutEntry->covm[14]) * lutEntry->pt;
   return val;
 }
+
```

Comment:
Hi @hscheid I don't get the change, is this intended?

Reply:
Hi @njacazio no. This was my mistake. Missed the one relevant commit.

Reply:
:)